Source/WebCore/svg/properties/SVGAnimationAdditiveValueFunctionImpl.cpp

```diff
@@ -35,8 +35,9 @@ namespace WebCore {
 Color SVGAnimationColorFunction::colorFromString(SVGElement& targetElement, const String& string)
 {
     static MainThreadNeverDestroyed<const AtomString> currentColor("currentColor"_s);
+    static MainThreadNeverDestroyed<const AtomString> currentcolor("currentcolor"_s);
 
-    if (string != currentColor.get())
+    if (string != currentColor.get() && string != currentcolor.get())
```

Comment:
r=me.

Reply:
Hmm, shouldn't it be case-insensitive then? Add a test for `currentCOLOR` and see what other implementations do?

Reply:
I agree with @annevk

Reply:
Note that you can also write `currentc\olor` in CSS. Should escape syntax work here? It being in SVG it might well not be defined I suppose...

Reply:
Firefox indeed seems to accept  case-insensitive currentcolor.
Chrome seems to as well, it is hard to say because invalid values also seem to fallback to currentcolor (unlike Firefox).

Reply:
equalLettersIgnoringASCIICase would work here

Reply:
We should test the value I mentioned above though. It would be good to confirm there's no need for a full blown CSS parser and then also assert that with a test.

Reply:
Indeed the "currentc\olor" case does not work. I wonder if this is the same problem https://bugs.webkit.org/show_bug.cgi?id=42871.

Reply:
@annevk how do you think should curre\ntcolor be treated? I do get a feeling the full CSS parser is needed.